RELEASE/scripts/autoscend.ash

```diff
@@ -452,6 +452,10 @@ boolean LX_calculateTheUniverse(boolean speculative)
 	{
 		return false;
 	}
+	if(get_property("_universeCalculated").to_int() >= min(3, get_property("skillLevel144").to_int()))
```

Comment:
Technically calculating the universe is only capped to 3 when in ronin or hardcore so anyone breaking ronin & still being in run might lose 1 or 2 potential casts of this but it doesn't really matter (anyone with 4 or 5 casts and going slow enough to break 1k turns of ronin deserves their fate).